core/cfn/ci_container_ti.dunfell.yml

```diff
@@ -0,0 +1,252 @@
+Description: >-
+    This template deploys an AWS CodeBuild project that
+    builds the docker container used for YP builds. It depends on the
+    CI Network template.
+
+Parameters:
+  NetworkStackName:
+    Description: >-
+        An environment name that is prefixed to resource names
+    Type: String
+
+  DockerhubSecretArn:
```

Comment:
We are inconsistent on passing a full ARN and synthesizing it off of stack names.

Reply:
Some things come from stacks, some don't.  For example, dockerhub secret doesn't come from a stack, so we have an arn.  Might be a doc item.